code/modules/projectiles/guns/projectile/automatic.dm

```diff
@@ -363,12 +367,13 @@
 	load_method = MAGAZINE
 	magazine_type = /obj/item/ammo_magazine/m762
 	allowed_magazines = list(/obj/item/ammo_magazine/m762, /obj/item/ammo_magazine/m762m)
+	projectile_type = /obj/item/projectile/bullet/rifle/a762
 
-	one_handed_penalty = 4
+//	one_handed_penalty = 45

```

Comment:
4*15=60?

Reply:
60 seems very, very high for what is essentially a shortened rifle. 45 is still decently punishing, and until any of these are actually activated I'm not sure it really matters, one way or another.